src/components/Publish/Pricing/index.tsx

```diff
@@ -28,6 +29,21 @@ export default function PricingFields(): ReactElement {
     type !== 'free' && setFieldValue('pricing.amountDataToken', 1000)
   }
 
+  // Always update everything when price value changes
+  useEffect(() => {
+    if (type === 'fixed' || type === 'free') return
+
+    const amountOcean =
+      isValidNumber(weightOnOcean) &&
+      isValidNumber(price) &&
+      isValidNumber(weightOnDataToken) &&
```

Comment:
why is `weightOnDataToken` checked for in this effect, and also in dependency array? I don't see it used in it at all

Reply:
right, it should be removed